src/main.ts

```diff
@@ -54,7 +45,8 @@ async function run() {
       }
 
       if (i > pollLimit) {
-        throw new StatusesError("Poll limit reached", statuses);
+        logStatuses(statuses);
```

Comment:
```suggestion
        process.stdout.write("\n");
        logStatuses(statuses);
```

Just in case we printed a newline-less `.` the last time around.